content/influxdb/cloud-dedicated/admin/custom-partitions/define-custom-partitions.md

```diff
@@ -0,0 +1,97 @@
+  to define custom partition strategies when creating a database or measurement.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Manage data partitioning
+weight: 202
+related:
+  - /influxdb/cloud-dedicated/reference/cli/influxctl/database/create/
+---
+
+Use the [`influxctl` CLI](/influxdb/cloud-dedicated/reference/cli/influxctl/)
+to define custom partition strategies when creating a database or measurement.
+By default, {{< product-name >}} partitions data by day.
+
+The partition strategy of a database or measurement is determined by a
+_partition template_ which defines the naming pattern for _partition keys_.
+Partition keys uniquely identify each partition.
+When a partition template is applied to a database, it becomes the default template
+for all measurements in that database, but can be overridden when creating a
+measurement.
+
+- [Create a database with a custom partition template](#create-a-database-with-a-custom-partition-template)
+- [Create a measurement with a custom partition template](#create-a-measurement-with-a-custom-partition-template)
+- [Example partition templates](#example-partition-templates)
+
+{{% note %}}
+#### Partition templates can only be applied on create
+
+You can only apply a partition template when creating a database or measurement.
+There is no way to update a partition template on an existing resource.
+{{% /note %}}
+
+Use the following command flags to identify
+[partition template parts](/influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/#tag-part-templates):
+
+- `--template-tag`: An [InfluxDB tag](/influxdb/cloud-dedicated/reference/glossary/#tag)
+  to use in the partition template.
+  _Supports up to 7 of these flags._
+- `--template-time`: A [Rust strftime date and time](/influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/#time-part-templates)
+  string that specifies the time format in the partition template and determines
+  the time interval to partition by.
+
+_View [partition template part restrictions](/influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/#restrictions)._
+  
+## Create a database with a custom partition template
+
+The following example creates a new `example-db` database and applies a partition
+template that partitions by two tags (`room` and `sensor-type`) and by hour using
+the time format `%Y-%m-%d %H:00`:
+
+```sh
+influxctl database create \
+  --template-tag room \
+  --template-tag sensor-type \
+  --template-time '%Y-%m-%d %H:00' \
+  example-db
+```
+
+## Create a measurement with a custom partition template
+
+The following example creates a new `example-meas` measurement and applies a
+partition template that partitions by two tags (`room` and `sensor-type`) and by
+week using the time format `%Y-%m-%d %H:00`:
+
+```sh
+influxctl measurement create \
+  --template-tag room \
+  --template-tag sensor-type \
+  --template-time '%Y-%m-%d %H:00' \
+  example-meas
```

Comment:
We probably will need to specify which database as well in this command.

Reply:
Sounds good. I've just left this as a placeholder until the command is in place.